shell/gpu/gpu_surface_gl.h

```diff
@@ -21,6 +22,14 @@ class GPUSurfaceGLDelegate {
   virtual bool GLContextPresent() = 0;
 
   virtual intptr_t GLContextFBO() const = 0;
+
+  virtual sk_sp<GrContext> GetGrContext() const {
```

Comment:
The pattern followed here for the rest of the methods was to make them pure virtual. Was a different approach taken for these two methods because you did not want to break iOS/Mac builds?